clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -757,7 +943,13 @@ LValue CIRGenFunction::buildBinaryOperatorLValue(const BinaryOperator *E) {
     LValue LV = buildLValue(E->getLHS());
 
     SourceLocRAIIObject Loc{*this, getLoc(E->getSourceRange())};
-    buildStoreThroughLValue(RV, LV);
+    if (LV.isBitField()) {
+      mlir::Value *val{0};
+      buildStoreThroughBitfieldLValue(RV, LV, val);
```

Comment:
Seems like the extra argument is pretty useless at this point? Please remove it!

Reply:
it's useless here, but is used in the other call of `buildStoreThroughBitfieldLValue`

Reply:
Gotcha, I missed the other one. See my comment in `buildStoreThroughBitfieldLValue`, please apply that change and just make this a bit more simple:

```
mlir::Value val;
buildStoreThroughBitfieldLValue(RV, LV, val);
```